src/vehicle.cpp

```diff
@@ -45,6 +45,7 @@
  */
 static const itype_id fuel_type_battery( "battery" );
 static const itype_id fuel_type_muscle( "muscle" );
+static const itype_id fuel_type_plutonium_cell ( "plut_cell" );
```

Comment:
You've brought my typo in:

```suggestion
static const itype_id fuel_type_plutonium_cell( "plut_cell" );
```

Reply:
Whoops, fixed